phi/tools/duckduckgo.py

```diff
@@ -33,6 +34,8 @@ def __init__(
         if news:
             self.register(self.duckduckgo_news)
 
+        self.verify: bool = verify
```

Comment:
A better param would be `verify_ssl`. What do you think?

Reply:
Makes sense. Will change.

Reply:
Done